scripts/install-chef-suse.sh

```diff
@@ -730,13 +730,11 @@ EOF
 fi
 
 chkconfig rabbitmq-server on
-ensure_service_running rabbitmq-server '^Node .+ with Pid [0-9]+: running'
+ensure_service_running rabbitmq-server
 
 chkconfig couchdb on
 ensure_service_running couchdb
 
-chmod o-rwx /etc/chef /etc/chef/{server,solr}.rb
```

Comment:
@vuntz do you think this caused jenkins to fail?

`WARN: Did not find config file: /etc/chef/solr.rb, using command line options`